specifications/api/swagger.yaml

```diff
@@ -559,6 +559,48 @@ x-transactionStatus: &transactionStatus
     - pending
     - in_ledger
 
+x-transactionMetadata: &transactionMetadata
+  description: |
+    Extra application data attached to the transaction.
+
+    Cardano allows users and developers to embed their own
+    authenticated metadata when submitting transactions.
+
+    The transaction metadata can be `null`, or a mapping from natural
+    numbers to metadata entries. These top-level keys can be any
+    number between `0` and `2^64 - 1`.
+
+    Note that metadata is not free-form JSON. Only a subset of JSON is
+    accepted as values. String values are limited in length, and
+    metadata is included in the overall transaction size limit.
+
+    See the example on the right to get an overview and the
+    equivalence table below to understand how metadata are mapped to
+    on-chain data types.
```

Comment:
Just out of interest, where does "on the right" refer to here?

Suggestion: if this wording relies on a particular rendering style, perhaps it's better to use a layout-neutral method to refer to examples? (I'm guessing that not everyone will be viewing the API spec through the web tool that we provide.)

Reply:
It refers to our current documentation layout indeed.